src/test/java/seedu/address/commons/util/FileUtilTest.java

```diff
@@ -20,4 +28,15 @@ public void isValidPath() {
         assertThrows(NullPointerException.class, () -> FileUtil.isValidPath(null));
     }
 
+    @Test
+    public void createFile_fileExists_success() throws IOException {
+        // valid path
+        Path validFilePath = testFolder.resolve("valid/file/path");
+
+        // create file at path
+        FileUtil.createFile(validFilePath);
```

Comment:
Is there a reason why you do this once here, and then assertDoesNotThrow for the same thing below?

Reply:
This test checks that the createFile command does not throw an error when the file already exists